CHANGELOG.md

```diff
@@ -56,6 +58,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Changed
 
 - [Component] What has been changed
+- [Button] Renewed Button component with theming support
 
 #### Fixed
 
```

Comment:
Changes to Accordion props (size enums) should also be listed? Are there other affected props in other components?

Reply:
Good point, missed it!